### PR TITLE
Release unofficial a Fedora RPM for obs-stufio on RPM Fusion

### PR DESCRIPTION
**Pull request by umireon**
Obs-studio is distributed on RPM Fusion so I think it is reasonable to ship our plugin with RPM.
We should ship our RPM as an unofficial or experimental distribution because the OBS project ships their obs-studio RPM as unofficial distribution.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -494,6 +494,39 @@ jobs:
...
+      image: "fedora:37"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        arch: [x86_64]
+    if: always()
+    needs: [clang_check]
+    steps:
+      - name: "Create an user"
```
Comment (royshil) [inline .github/workflows/main.yml]: nit: `Create a user`

**Review by royshil — APPROVED** [review]
LGTM
fix n ship

File: .github/workflows/main.yml (hunk 1 above)
Comment (umireon) [reply .github/workflows/main.yml]: Fixed! I'm bad at writing English 🤣

**Comment by umireon** [comment]
@royshil Can you add documentation about this RPM after this PR is merged? Or do you want me to add a note?

**Pull request merged**